### ISSUE TITLE
Learn Greetings in your First Day at the Office - Task 24 - Description doesn't match audio

### ISSUE THREAD

**Issue by gikf** [freeCodeCamp/freeCodeCamp#52708]
### Describe the Issue

Description and question refers to the _Here's the main office area. And over there, let me show you, that's the conference room._ text. However the audio mentions _let me show you_ in context of showing to the desk.

### Affected Page

https://www.freecodecamp.org/learn/a2-english-for-developers/learn-greetings-in-your-first-day-at-the-office/task-24

### Your code

n/a

### Expected behavior

Audio used in task should match the description context. Or the other way around.

### System

n/a

**Comment by nieldakarla** [comment]
Yes, this is a problem. None of the answers are correct. Here's how I'd fix it:
change the option "the conference room" to "the desk". This should solve the problem. 
We could also:
- simplify the question to "What is Maria directing attention to?"
- simplify the example to "Here's the main office area. Let me show you to the conference room."

**Comment by moT01** [comment]
I am fine with the suggestion. 

You could use `This is the main office area. Let me show you to the conference room.` as the example. What do you think @nieldakarla?

**Comment by nieldakarla** [comment]
Agreed @moT01! I'm going to add the help wanted label.
